blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -89,401 +33,93 @@ export class RunDetailsPipeline extends Component {
             document.addEventListener('keydown', this._handleKeys, false);
```

Comment:
I just did something like this for the setup wizard, i think you just need to listen for scroll events on the scroll area and when following along maybe set a flag so those are ignored. This was thanks to @cliffmeyers 's awesome suggestion but seems to work well: https://github.com/jenkinsci/jenkins/pull/2825/files#diff-1b14e0dbd9ab104cdbcea2c1f823bcacR473